run.py

```diff
@@ -47,7 +51,7 @@ def __init__(self, options_bin):
                 'quiet': True,
                 'noplaylist': True,
                 'format': 'bestaudio/best',
-                'keepvideo': True,
+                # 'keepvideo': True,
```

Comment:
Feel free to remove comments. If we need to revert the code in the future, we can always revisit the git hisory.